src/main/java/de/lh/tool/domain/model/ItemHistory.java

```diff
@@ -39,9 +37,8 @@
 	@JoinColumn(insertable = true, name = "user_id", updatable = true)
 	private User user;
 
-	@Column(name = "timestamp", nullable = false)
-	@Temporal(TemporalType.DATE)
-	private Date timestamp;
+	@Column(nullable = false)
+	private LocalDateTime timestamp;
```

Comment:
Hast du überlegt, ob man hier ZonedDateTime möglicherweise besser wäre?
Ich denke LocalDateTime ist in Ordnungn, aber lieber nochmal nachgefragt ;)